test/kino_explorer/data_transform_cell_test.exs

```diff
@@ -1631,7 +1631,7 @@ defmodule KinoExplorer.DataTransformCellTest do
             "team" => :string,
             "weekday" => :string
           },
-          "group_by" => ["weekday"],
+          "columns" => ["weekday"],
```

Comment:
Is this change backwards compatible? Could it mean that we now can no longer load previous cells?

Reply:
Yep! It is! There’s a normalize function to update the attrs